lib/cookbook_sdk/rake_tasks/provisioning_tasks.rb

```diff
@@ -56,21 +92,20 @@ def create_custom_client_rb(target_folder)
       while (buff = input.read(4096))
         output.write(buff)
       end
-      output.write("
-# To enable chef-solo without sudo.
-# https://docs.chef.io/ctl_chef_client.html#run-as-non-root-user
-cache_path '#{Dir.pwd}/#{target_folder}/.chef'
-")
+
+      output.write(cache_path_config(target_folder))
+
+      config = read_configuration(configuration_file)
```

Comment:
enclose this with a begin/rescue statement and it takes care of file not found and parsing errors - set `config[:handlers]` as empty of any exception is triggered.